Podfile

```diff
@@ -1,5 +1,5 @@
 # Uncomment the next line to define a global platform for your project
-platform :ios, '11.0'
```

Comment:
Can you update the Podfile to point to the dev-5.0.0 branches on the dependencies? 
Also add an explicit dependency to AEPRulesEngine to avoid build failures.
```
pod 'AEPRulesEngine', :git => 'https://github.com/adobe/aepsdk-rulesengine-ios.git', :branch => 'dev-v5.0.0'```